### PR TITLE
I created a menu bar, added a photo, added an introduction, added social media Icons and text for a blog post.

### PR DESCRIPTION
**Pull request by JoachimIsaac**
Did some of this before our meeting, will keep the commits smaller and more frequent moving forward.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -3,16 +3,119 @@
...
+        while learning data structures and algorithms.
+        </p>
+        <p>
+        After learning data structures and algorithms, you could also try teaching them to your friends or 
+        people who are generally looking to learn them. I have been recently doing videos on solving programming problems 
+        on youtube using javascript. Explaining hard concepts to people reinforces your knowledge and challenges you to 
+        really understand the material.
+        </p>
+        <h2>Here is a Video of me answering a Programming Question with javascript</h2>
+        //Iframe of video here.
```
Comment (stekolla) [inline portfolio/src/main/webapp/index.html]: Comments in HTML can be done like the following:
```suggestion
        <!-- Iframe of video here. -->
```

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -3,16 +3,119 @@
...
+        people's lives easier.
+      </p>
+   
+     
+      <p class="indicator">Click Here to find out one of my interests</p>
+      <div class="button-container"><button onclick="addRandomGreeting()">Click</button></div>
+      <div id="facts-container"></div>
     </div>
+
+    <section id="projects"> </section>
```
Comment (stekolla) [inline portfolio/src/main/webapp/index.html:42]: Nice use of semantic elements!

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -3,16 +3,119 @@
...
+    <section id="social_media">
+        <a href="https://www.w3schools.com"><i class="fab fa-linkedin"></i></a>
+        <a href="https://www.w3schools.com"><i class="fab fa-github-square"></i></a>
+        <a href="https://www.w3schools.com"><i class="fab fa-youtube-square"></i></a>
```
Comment (stekolla) [inline portfolio/src/main/webapp/index.html]: Update these links to point to the correct URLs?

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -16,13 +16,14 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const facts =
+      ['I have lost 150lbs in my life time', 'I love running and hope to do a marathon soon'
```
Comment (stekolla) [inline portfolio/src/main/webapp/script.js]: It might be easier to keep track of each item in this list if it's on its own line.

Most companies will usually have a style guide to help you format your code, and they'll usually cover a wide range of topics.

Here's Google's external JavaScript style guide as an example: https://google.github.io/styleguide/jsguide.html

And it recommends using clang-format for this kind of formatting: https://clang.llvm.org/docs/ClangFormat.html

File: portfolio/src/main/webapp/WEB-INF/index.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```
Comment (stekolla) [inline portfolio/src/main/webapp/WEB-INF/index.html]: Is this file necessary since you expanded the other one?

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -3,16 +3,119 @@
...
+      <p class="introduction">
+        My name is Joachim Isaac, I am a Software engineer by trade, an avid amateur boxer, weightlifter and stock trader.
+        I am very passionate about technology - not a specific framework or language.
+        I'm interested in web developement and artificial intellegence to create a cool web applications, that make
+        people's lives easier.
+      </p>
+   
+     
+      <p class="indicator">Click Here to find out one of my interests</p>
+      <div class="button-container"><button onclick="addRandomGreeting()">Click</button></div>
```
Comment (stekolla) [inline portfolio/src/main/webapp/index.html:38]: Inline event handlers like this onclick="..." can be done for simplicity, but you'll also want to learn how to add them directly from scripts.  There's no need to change this now, but take a look at the following article when you have time: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events

**Review by stekolla — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (JoachimIsaac) [reply portfolio/src/main/webapp/index.html:42]: Thanks Steve!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (JoachimIsaac) [reply portfolio/src/main/webapp/index.html]: Yeah, sorry about that. 
Is it an acceptable standard to use this as a place holder? 
<a href="#">

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (JoachimIsaac) [reply portfolio/src/main/webapp/script.js]: ok, I formatted the array neatly using the google style guide.

 I tried to install homebrew to install clang but had no luck. Then I realized you just wanted me to keep my code formatting consistent with the clang-format. Rather than actually downloading it.

File: portfolio/src/main/webapp/WEB-INF/index.html (hunk 5 above)
Comment (JoachimIsaac) [reply portfolio/src/main/webapp/WEB-INF/index.html]: Not sure how I ended up having two. Deleted the old one though.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (stekolla) [reply portfolio/src/main/webapp/index.html]: Yeah, no worries.  :)

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (stekolla) [reply portfolio/src/main/webapp/script.js]: Looks good to me.  I'll do some research to see if I can figure out how to easily enable clang-format for the cloud shell editor.

**Review by stekolla — APPROVED** [review]
